### DISCUSSION TITLE
Unable to select Download location on GoogleTV

### DISCUSSION THREAD

**Discussion by akaGrade0ne** [localsend/localsend#1449] [Discussion (English, International)]
As per the title, I'm unable to open the folder selection dialog in LocalSend to actually choose where I want files to go. When I click on 'destination' in the settings menu on the TV nothing happens. I want to be able to set the destination to an external drive that I have connected to the TV as the TV itself has a paltry amount of space available (sub 400MB). The drive is visible and accessible via alternative file exploring apps. 

I believe the issue might be due to permissions, as I'm unable to even open the 'destinations' dialog, so I can't even change the location to another folder of locally connected storage.

Any help would be much appreciated, thank you.

LocalSend 1.13.1 (42)
Sony A80J ATV OS10
Kernel 4.19.75

**Comment by ddmplaya** [comment]
Me pasa lo mismo, no puedo modificar la carpeta de descargas en la aplicación instalada en mi TV. Si encuentro una solución te la comparto.
